doc/dbus.rst

```diff
@@ -244,6 +244,25 @@ org.freedesktop.ratbag1.Profile
         Provides the object paths of all LEDs in this profile, see
 	:ref:`led`.
 
+.. attribute:: ReportRate
+
+        :type: u
+        :flags: read-write, mutable
+
+        uint for the report rate in Hz assigned to this profile
+
+.. attribute:: ReportRates
```

Comment:
Isn't this a device "capability"? I'm assuming that there is no device that has a different number of report rates or different values for report rates per profile?

Reply:
true, but for simplicity I think it's better to keep those two together than spreading them across multiple objects.